### DISCUSSION TITLE
Multi-factor authentication for SSH and Kubernetes

### DISCUSSION THREAD

**Discussion by klizhentas** [gravitational/teleport#6166] [Show and tell]
One of the coolest features in upcoming 6.1 is per-session multi-factor authentication for Kubernetes and SSH. Here is a demo from our lead engineer and author of this feature Andrew: 

[YouTube demo video](https://www.youtube.com/watch?v=74g0V2XwbE4)

**Comment by travelton** [comment]
Awesome! Are there plans to extend this functionality to DB access?
